src/Microsoft.Health.Fhir.Core.UnitTests/Features/Security/RoleLoaderTests.cs

```diff
@@ -206,13 +208,13 @@ public void GivenValidDataActions_WhenSpecifiedAsRoleActions_AreRecognized()
 
         [Theory]
         [MemberData(nameof(GetInvalidRoles))]
-        public void GivenInvalidRoles_WhenLoaded_RaiseValidationErrors(string description, object roles)
+        public async Task GivenInvalidRoles_WhenLoaded_RaiseValidationErrors(string description, object roles)
         {
             Assert.NotEmpty(description);
-            Assert.Throws<InvalidDefinitionException>(() => Load(roles));
+            await Assert.ThrowsAsync<InvalidDefinitionException>(() => Load(roles));
         }
 
-        private static AuthorizationConfiguration Load(object roles)
+        private static async Task<AuthorizationConfiguration> Load(object roles)
```

Comment:
A crude regex search (`async\s+Task\S*\s+(?![^_(]*_)(?![^(]+Async\()[^(]+\(`) suggest there might be something like 193 such methods. I'm not sure this is really all that necessary. One convention is to add Async only when there is a synchronous version of the method. But we're not very consistent (sometimes we use the suffix, sometimes we don't)

Reply:
I see the asp.net team prefers the suffix: https://github.com/dotnet/aspnetcore/wiki/Engineering-guidelines#async-method-patterns

Reply:
I agree that it didn't make sense to rename 193 methods in this PR!

To your point on consistency - it might be nice to create a story to look into making our async naming a bit more uniform (whether that's following the asp.net team's standard or only adding "async" if there is a synchronous version). Thoughts?